packages/sources/react/src/components/VtmnTooltip/VtmnTooltip.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import { VtmnTooltipPosition } from './types';
+import { VtmnIcon } from '../VtmnIcon';
+import { VitamixId } from '../../../../icons/dist/vitamix/font/vitamix';
+
+export interface VtmnTooltipProps
+  extends React.ComponentPropsWithoutRef<'span'> {
+  /**
+   * The position of tooltip from original element.
+   * @default 'top'
+   */
+  position?: VtmnTooltipPosition;
+
+  /**
+   * The tooltip text of the span.
+   * @default ''
+   */
+  tooltip?: string;
+
+  /**
+   * The tooltip associated icon.
+   * @default ''
+   */
+  icon?: VitamixId;
+
+  /**
+   * The hoverable text.
+   * @default ''
+   */
+  textValue?: string;
```

Comment:
Should be required no ?

Reply:
Not this one, if you want to display an icon this field is not required, and it will overwritten by the icon if the icon is displayed :)